src/components/Edit/ChangeSummary.vue

```diff
@@ -118,7 +123,8 @@ export default class ChangeSummary extends Mixins(
 
   /** The firm type label. */
   get firmLabel (): string {
-    return this.getResource.changeData.orgPersonInfo.orgPersonLabel
+    let thisLabel = this.getResource.changeData.orgPersonInfo.orgPersonLabel
+    return thisLabel === 'Partners' ? 'Partner' : thisLabel
```

Comment:
I find this hard to understand.

Is this only to reuse an existing resource label but change Partners to Partner?

Hmm, it's too bad the UI design shows a different label on the summary page than on the first page.

I don't think you should try to reuse the original label. I think you should use a new resource label for this (eg, "Partner Information").

Reply:
I don't know what `this.getResource.changeData.orgPersonInfo.orgPersonLabel` will be other than "Partners", but I am guessing it will be a number of options. Since this label is only used in one place -- [here](https://github.com/bcgov/business-edit-ui/blob/a791dd35e57fd4b2714fc191f8a995257263f595/src/components/Edit/ChangeSummary.vue#L60) -- I change this label only when `this.getResource.changeData.orgPersonInfo.orgPersonLabel` is "Partners".

Reply:
Yes, it is only to reuse an existing resource label but change Partners to Partner.

Reply:
I don't like reusing/modifying a resource string that is used for another purpose.

However, since ChangeSummary is only used for firm changes, what do you think of keeping this really simple and just using a ternary in the template to toggle between Partner/Proprietor?

Reply:
If it's only between Partner and Proprietor, then we can use conditional rendering in html elements.